eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java

```diff
@@ -135,7 +109,7 @@ public void exec() throws IOException {
             }
             int count = 0;
             for (final String name : names) {
-                if (!this.objectionary.contains(name)) {
+                if (!this.objectionaryByHash(this.hsh).contains(name)) {
```

Comment:
@maxonfjvipon Looks like we can add `contains()` method to `Objectionaries` interface. In this case  we will be able to encapsulate the logic of the `objectionaryByHash` inside `Objectionaries` implementation (and apparently we won't need `with` method).
In other words, I suggest treating `Objectionaries` as a living object and showing it respect, not merely treating it as a bag of `Objectionary` objects.

https://www.yegor256.com/2014/11/20/seven-virtues-of-good-object.html#1-he-exists-in-real-life:
> Moreover, an object should be [anthropomorphized](https://en.wikipedia.org/wiki/Anthropomorphism), i.e. treated like a human being (or a pet, if you like them more). By this I basically mean that an object is not a data structure or a collection of functions.

Reply:
@volodya-lombrozo The problem is here that `ProbeMojo` and `PullMojo` where `Objectionaries` is used - require different objectionaries to insert into `Objectionaries`. So we can't just let `Objectionaries` put any objectionary it wants, we need to give the objectionary to it from the outside.
In your case `contains` will look like 
```java
bool contains(CommitHash hash, String name, Objectionary obj)
```
Here `hash` - key in map, `name` - name of object, `obj` - objectionary to put if it's absent with given `hash`
So I don't think it's a good idea

Reply:
@maxonfjvipon Can we create `Objectionaries` by using a constructor, instead of `with` method? I don't understand this: 
> The problem is here that ProbeMojo and PullMojo where Objectionaries is used - require different objectionaries to insert into Objectionaries 

Why we can't create `Objectionaries` by using constructor (and provide all the required `Objectionary` instances) and then just call `contains()` on it? Why do we need to `get()` it and then call `contains()`?